distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerProxy.java

```diff
@@ -38,8 +42,15 @@ public ServiceManagerProxy(ServiceManager serviceManager) {
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-
-        Subject subject = SECURITY.runAsAdmin(SECURITY::getSystemSubject);
-        return subject.execute(() -> method.invoke(serviceManager, args));
+        final RetryPolicy retryPolicy = new RetryPolicy().retryOn(NullPointerException.class)
+                .withDelay(1, TimeUnit.SECONDS)
+                .withMaxRetries(10);
```

Comment:
Wonder if we might need a higher number for very slow systems. Probably ok as is.